app/models/recommendation.rb

```diff
@@ -25,6 +25,52 @@ def validate(record)
 
   before_validation :set_pending_status
 
+  def self.create_by_id_and_email(recommender, params)
```

Comment:
This should not be taking the `params` parameter. You should pass in `recommendee_ids` and `recommendee_emails` separately. Parsing params is the controller's job.